src/Core/Block.h

```diff
@@ -26,7 +26,7 @@ class Block
 {
 private:
     using Container = ColumnsWithTypeAndName;
-    using IndexByName = std::unordered_map<String, size_t>;
+    using IndexByName = std::unordered_multimap<String, size_t>;
```

Comment:
Argh, multimap is very slow - should replace with smth like `unordered_map<String, vector<size_t>>`